src/librustdoc/html/render/mod.rs

```diff
@@ -3413,7 +3413,14 @@ fn render_struct(
                 match field.kind {
                     clean::StrippedItem(box clean::StructFieldItem(..)) => write!(w, "_"),
                     clean::StructFieldItem(ref ty) => {
-                        write!(w, "{}{}", field.visibility.print_with_space(cx.tcx()), ty.print())
+                        write!(
+                            w,
+                            "{}{}",
+                            field
+                                .visibility
+                                .print_with_space(cx.tcx(), field.def_id),
+                            ty.print()
+                        )
```

Comment:
nit: rustfmt really mangles this, can you use a local variable instead?
```suggestion
                        let vis = field.visibility.print_with_space(cx.tcx(), field.def_id);
                        write!(w, "{}{}", vis, ty.print());
```

Reply:
The current version looks like this:

```rust
                        write!(
                            w,
                            "{}{}",
                            field.visibility.print_with_space(cx.tcx(), field.def_id),
                            ty.print()
                        )
```

Is that okay?

Reply:
Not my preference but good enough not to worry about.